admin_guide/managing_networking.adoc

```diff
@@ -1379,6 +1379,11 @@ means that egress policy types, IPBlock, and combining `podSelector` and
 `namespaceSelector` are not available in {product-title}.
 ====
 
+[NOTE]
+====
+It is not recommended to apply *ovs-networkpolicy* features on default {product-title} projects, as it can disrupt communication with other pods.
```

Comment:
>Do not apply *ovs-networkpolicy* features to default {product-title} projects because they can disrupt communication with other pods.

Reply:
Fixed! I included some input from the bug reporter as well, as it seems any of the  `NetworkPolicy` suite is not wise on default projects. Thanks!